source/docs/articles/sites/code/applying-upstream-updates.md

```diff
@@ -40,29 +26,36 @@ If you have overwritten core, see [Undo Git Commits](/docs/articles/local/undo-g
 5. Deploy the upstream updates to your Test Environment by clicking **Pull (Content from Live and) Code from Development** in the Code workspace in the Test Environment.
 6. Click **Visit Testing Environment** to test the update against your Live site’s content base.
 7. Deploy the upstream updates to your Live Environment by clicking **Pull Code from Testing** in the Code workspace in the Live environment.
-![The Code tab in a Pantheon site's dashboard's Live environment](/source/docs/assets/images/desk_images/357432.png)
 8. Click **Visit Live Environment** to verify the update is live.
 ![The Code tab in the Pantheon site's dashboard's Live Environment, showing the upstream commits in the log as deployed.](/source/docs/assets/images/desk_images/357435.png)
 
+## About Core Updates

```

Comment:
The user reads 1. how to prepare for automatic updates, 2. how to complete automatic updates, and 3. definition for automatic updates  - seems a bit strange to define the task after completing it. 

I agree we should make it less visually intimidating, but IMO this doesn't seem like an ideal content flow.